src/sys/socket/mod.rs

```diff
@@ -779,6 +798,18 @@ pub enum ControlMessageOwned {
     #[cfg(any(target_os = "android", target_os = "linux"))]
     #[cfg_attr(docsrs, doc(cfg(all())))]
     ScmTimestampns(TimeSpec),
+    /// Realtime clock timestamp
+    ///
+    /// [Further reading](https://man.freebsd.org/cgi/man.cgi?setsockopt)
+    #[cfg(target_os = "freebsd")]
+    #[cfg_attr(docsrs, doc(cfg(all())))]
```

Comment:
Please remove the 2 newly-added `#[cfg_attr(docsrs, doc(cfg(all())))]` attributes, as [we are doing this to the codebase](https://github.com/nix-rust/nix/pull/2192)

Reply:
All fixed!